dupcop

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env node
+'use strict';
```

Comment:
`'use strict'` can be omitted in ES6 modules and if you have only `class`es in CommonJS module.